source/InterfaceLayer.cpp

```diff
@@ -42,14 +74,85 @@ void InterfaceLayer::OnImGuiRender()
     }
     ImGui::SameLine();
     ImGui::Text("counter = %d", counter);
-
     ImGui::End();
-    ImGui::SetNextWindowPos(ImVec2(10, 200), ImGuiCond_Always);
+    }
+  
+    
+    
+    // Works fine without it
+
+    // int sdl_width = 300, sdl_height = 300, controls_width = 0;
+    // controls_width = sdl_width;
+    // if ((controls_width /= 3) < 300) { controls_width = 300; }
+
+
+    // ImGui::SetNextWindowPos(ImVec2(10, 10), ImGuiCond_Always);
+    // ImGui::SetNextWindowSize(
+    //     ImVec2(static_cast<float>(controls_width), static_cast<float>(sdl_height - 20)),
+    //     ImGuiCond_Always
+    //     );
+
+    
+
+    ImGui::SetNextWindowPos(ImVec2(50,920));
```

Comment:
It is better not to use constants in positioning. If a window would be too small, it won't be visible